bindings/python/py_src/tokenizers/__init__.pyi

```diff
@@ -275,6 +275,21 @@ class Encoding:
         """
         pass
     @property
+    def n_sequences(self) -> int:
+        """The number of sequences represented
+
+        Returns:
+            :obj:`int`: The number of sequences in this :class:`~tokenizers.Encoding`
+        """
+        pass
+    def set_sequence_id(self, sequence_index: int):
```

Comment:
Is that really something we need to expose ?

Reply:
Yes, it's just for consistency. The user can manually merge multiple `Encoding`s, and this allows to manually set the sequence id for each of them.